src/state/opamFormatUpgrade.ml

```diff
@@ -19,6 +19,8 @@ let slog = OpamConsole.slog
 
 exception Upgrade_done of OpamFile.Config.t
 
+let v2_0 = OpamVersion.of_string "2.0"
```

Comment:
`v2_0` defined twice (cf. [L1042](https://github.com/ocaml/opam/blob/16ed21c0dd262a57afbff827e42738a93355d09b/src/state/opamFormatUpgrade.ml#L1042)). By convention, at the beginning of the files these versions are not defined ([for ex.](https://github.com/ocaml/opam/blob/16ed21c0dd262a57afbff827e42738a93355d09b/src/state/opamFormatUpgrade.ml#L332)) and defined only in front of their translation functions.

Reply:
This is fixed - although for some reason GitHub is still display this line, even though 3c6ddbe1 doesn't have it!

Reply:
reintroduced by 5e4798f6 ^^

Reply:
🤦‍♂